### PR TITLE
chore: replace antd divider

### PR DESCRIPTION
**Pull request by daibhin**
## Problem

We are still using `Divider` from antd in some places

## Changes

|Before|After|
|----|----|
| b-cohort | a-cohort |

|Before|After|
|----|----|
| before | after |

|Before|After|
|----|----|
| Screenshot 2024-01-24 at 14 01 12 | Screenshot 2024-01-24 at 14 21 47 |
| | Minus is just an artefact of faking data locally |

## How did you test this code?

Tested where used as seen in screenshots

### CODE REVIEW

File: frontend/__snapshots__/filters-taxonomic-filter--actions--light.png
Comment (thmsobrmlr) [inline frontend/__snapshots__/filters-taxonomic-filter--actions--light.png:1]: The lighter divider was more harmonious here (in the light theme).

**Review by thmsobrmlr — COMMENTED** [review]
Thanks for the cleanup! 🙌 

I think the lighter divider was better (see screenshots). Any reason not to use this color in light mode?

File: frontend/__snapshots__/filters-taxonomic-filter--actions--light.png
Comment (daibhin) [reply frontend/__snapshots__/filters-taxonomic-filter--actions--light.png:1]: No particular reason - I think it was a custom antd color so I just went with the default divider color. Might be worth customizing it for this particular use case

**Review by thmsobrmlr — APPROVED** [review]
Thanks!

**Pull request merged**